packages/api-explorer/src/components/JsonViewer/index.jsx

```diff
@@ -23,11 +24,15 @@ export default function JsonViewer({ schema, missingMessage }) {
         whiteSpace: 'pre-wrap',
         wordBreak: 'break-all'
       }}
-    /> : <FormattedMessage id={missingMessage} />
+    /> : message
   )
 }
 
 JsonViewer.propTypes = {
-  schema: PropTypes.object.isRequired,
-  missingMessage: PropTypes.string.isRequired
+  schema: PropTypes.oneOfType([
+    PropTypes.object,
+    PropTypes.array
+  ]).isRequired,
+  // eslint-disable-next-line react/require-default-props
+  missingMessage: PropTypes.string
```

Comment:
Why could not render a message if schema is not present. I suppose a message should be visible

Reply:
I have restored this piece of component